organization/pull_requests/review_process.rst

```diff
@@ -1,57 +1,34 @@
 Review and merge process
 ========================
 
-.. note::
-
-    This page is intended to provide insight into the pull request (PR) review
-    process that we aspire to. As such, it is primarily targeted at engine
-    maintainers who are responsible for reviewing and approving pull requests.
-    That being said, much of the content is useful for prospective contributors
-    wanting to know how to ensure that their PR is merged.
-
 From a high level, the ideal life cycle of a pull request looks like the
 following:
 
-  1. A contributor opens a PR that fixes a specific problem (optimally closing
-     a GitHub `issue <https://github.com/godotengine/godot>`_ or implementing
-     a `proposal <https://github.com/godotengine/godot-proposals>`_).
+  1. A contributor :ref:`opens a pull request <doc_pr_workflow>` that fixes a
+     specific problem (ideally closing a GitHub `issue <https://github.com/godotengine/godot>`_
+     or implementing a `proposal <https://github.com/godotengine/godot-proposals>`_).
 
-  2. Other contributors provide feedback on the PR (including reviewing and/or
-     approving the PR, as appropriate).
+  2. The :ref:`bugsquad and triage team <doc_areas>` **categorize** the pull request,
+     adding appropriate tags and requesting reviews from :ref:`area maintainers <doc_areas>`.
 
-  3. An engine maintainer reviews the code and provides feedback, requests
-     changes, or approves the pull request, as appropriate.
+  3. Contributors discuss **whether the approach of the PR is appropriate** to fix the problem
+     at hand, and leave feedback. They may also approve the pull request at this stage, noting
+     that they did not review the code.
```

Comment:
Currently it's pretty rare for contributors (at least folks with some merged contributions) to do a formal approval on GH just on the principle of a PR but without a code review.

I agree that it's good to have that assessment done first, but since currently it's not usually expressed via the "Approve" button, we should consider whether we're asking to change this workflow (contributors should now approve and say what they're approving), or propose a different way of confirming that a PR is wanted.

My main worry with GH approvals is that most people will skip writing any detailed info about what they reviewed, and the release management team will have to guess whether contributor X actually reviewed both the approach and the code, or just the approach, or just the code.